core/src/test/java/com/sequenceiq/cloudbreak/service/stack/flow/StackOperationServiceTest.java

```diff
@@ -118,9 +118,6 @@
     @Mock
     private TargetedUpscaleSupportService targetedUpscaleSupportService;
 
-    @Mock
-    private UpdateNodeCountValidator updateNodeCountValidator;
-
     @Spy
     private UpdateNodeCountValidator updateNodeCountValidator = new UpdateNodeCountValidator();
```

Comment:
This is an interesting one. Because of double initialization, the unit test was not failing. It failed the checkstyle test, upon fixing it revealed the test case issue.